LBH.AdultSocialCare.Api/V1/Infrastructure/Entities/Package.cs

```diff
@@ -44,6 +44,6 @@ public class Package
         /// <summary>
         /// Gets or sets the Date Updated
         /// </summary>
-        public DateTime? DateUpdated { get; set; }
+        public DateTimeOffset? DateUpdated { get; set; }
```

Comment:
Not a change request, but this field can be attributed with `[DatabaseGenerated(DatabaseGeneratedOption.Identity)]` to receive a value on insert/update automatically